### PR TITLE
Issue 13 brightness contrast

### PR DESCRIPTION
**Pull request by abhidg**
Added brightness and controls to menu using CSS3 filters.

### CODE REVIEW

File: src/app.jsx [hunk 1]
```diff
@@ -32,9 +32,10 @@ export default class App extends React.Component {
         }}
       />
       <Viewer 
-        onOpen={this.onOpen.bind(this)}
+        onOpen={this.onOpen.bind(this)} ref={
```
Comment (martinjrobins) [inline src/app.jsx:35]: Do you know how this ref attribute works? I know I put it in first, but that was just copying an example, be good if you can explain it to me!

**Review by martinjrobins — APPROVED** [review]
This looks great, thanks @abhidg. Can you also left justify the labels on the buttons as well, so the icons on the left line up?

File: src/app.jsx (hunk 1 above)
Comment (abhidg) [reply src/app.jsx:35]: It's to bind `this` - which is not bound in class methods by default, can be bypassed by using arrow functions or some new syntax, https://reactjs.org/docs/handling-events.html

Comment (martinjrobins) [reply src/app.jsx:35]: No, not the `.bind(this)`, the `ref = {viewer => {this.viewer = viewer;}}`. I can see that it gives the parent component access to the child component, but I don't understand what the the `ref` attribute is and why this callback needs to be assigned to it

Comment (abhidg) [reply src/app.jsx:35]: https://reactjs.org/docs/refs-and-the-dom.html#callback-refs to access the viewer DOM node I presume

**Pull request merged**